autoload/ctrlsf/backend.vim

```diff
@@ -83,6 +83,7 @@ let s:backend_args_map = {
 " BuildCommand()
 "
 func! s:BuildCommand(args, for_shell) abort
+    let g:ctrlsf_pwd = getcwd()
```

Comment:
Avoid using global variables as they're typically reserved for options. Instead, define it as `s:current_pwd` (similar to `s:current_mode`), and add a new public method `ctrlsf#GetFilePath(path)`.